README.md

```diff
@@ -10,7 +10,15 @@ Download/clone the code from this repo then update the configuration.
 
 The project needs to be configured with your Authgear client application's **Endpoint** and **Client ID** in order for the authentication flow to work.
 
-To do this, in `App.tsx`, replace `"<CLIENT_ID>"`, `"<AUTHGEAR_ENDPOINT>"` with with your own Authgear client application credentials.
+To do this, create a `.env` file with the following keys in the root directory of the project:
+```
+AUTHGEAR_CLIENT_ID=
+AUTHGEAR_ENDPOINT=
+AUTHGEAR_REDIRECT_URI=com.authgear.example.rn://host/path
+```
+Add the values for your own Authgear client application credentials in the corresponding key.
+
+Or rename `.env.example` to `.env` then add the Client ID and Endpoint for your Authgear Client Application.
```

Comment:
The convention is calling it `.env.template`, and not renaming it but copy the values into `.env`. Renaming it would mess with the version control